packages/charts/chart-web-components/src/donut-chart/donut-chart.styles.ts

```diff
@@ -28,10 +28,10 @@ import {
 export const styles = css`
   ${display('inline-block')}
 
-  .root {
+  :host {
     ${typographyBody1Styles}
     align-items: center;
-    display: flex;
+    // display: flex;
```

Comment:
remove?

Reply:
will uncomment it once I add handlers for width and height attributes